The link interface of target "TBB::tbb" contains:  Threads::Threads  but the target was not found.

**build from:** https://github.com/oneapi-src/oneTBB/archive/refs/tags/v2021.9.0.tar.gz

**usage:**
```cmake
find_package(TBB REQUIRED)
target_link_libraries(mold PRIVATE TBB::tbb)
```

**reported messages:**
```cmake
CMake Error at /home/leleliu008/.ndk-pkg/installed/android-21/arm64-v8a/tbb/lib/cmake/TBB/TBBTargets.cmake:61 (set_target_properties):
  The link interface of target "TBB::tbb" contains:

    Threads::Threads

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /home/leleliu008/.ndk-pkg/installed/android-21/arm64-v8a/tbb/lib/cmake/TBB/TBBConfig.cmake:1 (include)
  CMakeLists.txt:184 (find_package)
```

**reason:** It seems that `find_package(Threads REQUIRED)` is missing in `TBBTargets.cmake`

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -265,6 +265,10 @@ else()
                 COMPONENT devel)
         file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
                    "include(\${CMAKE_CURRENT_LIST_DIR}/${PROJECT_NAME}Targets.cmake)\n")
+        if (NOT BUILD_SHARED_LIBS)
+            file(APPEND ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
+                       "include(CMakeFindDependencyMacro)\nfind_dependency(Threads)\n")
+        endif()
 
         write_basic_package_version_file("${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake"
                                          COMPATIBILITY AnyNewerVersion)

